src/app/view/components/_common/dot-icon/dot-icon.component.ts

```diff
@@ -1,5 +1,11 @@
 import { Component, Input } from '@angular/core';
 
+enum DotIconSize {
```

Comment:
shouldn't this be on `_varialbles.scss`

Reply:
Is a enum, can't be in scss code.

Reply:
![image](https://user-images.githubusercontent.com/37185433/63465106-bd44a900-c41d-11e9-8174-dff7b01449c2.png)

shouldn't it match sizes with scss?